releasenotes/notes/fix-configurable-fake-backend-6a07ca5a6159baf5.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    The member ``name`` of :py:class:`~qiskit.test.mock.utils.configurable_backend.\
+    ConfigurableFakeBackend` is updated to a class method. This used to cause a bug
```

Comment:
```suggestion
    The member ``name`` of ``qiskit.test.mock.utils.ConfigurableFakeBackend``
     has been changed to ``backend_name``. This was done to avoid a conflict with the
     ``name()`` method inherited from the parent abstract
     :class:`~qiskit.providers.BackendV1` class. This makes
     ``ConfigurableFakeBackend`` compatible with anything expecting a
     :class:`~qiskit.providers.BackendV1` object. However, if you were using the
     ``name`` attribute directly before you will now need to either call it as a method or
     access the ``backend_name`` attribute instead.
```